### PR TITLE
upload annotations + multi-thread upload + resume

### PR DESCRIPTION
**Pull request by Renthal**

### CODE REVIEW

File: darwin/client.py [hunk 1]
```diff
@@ -77,7 +77,12 @@ def get(self, endpoint: str, retry: bool = False, raw: bool = False):
             return self.get(endpoint=endpoint, retry=False)
 
         if response.status_code != 200:
-            print("TODO, fix me get", response.json(), response.status_code)
+            print(
+                f"Client get request response ({response.json()}) with unexpected status "
+                f"({response.status_code}). "
+                f"Client: ({self})"
+                f"Request: (endpoint={endpoint})"
+            )
```
Comment (andreaazzini) [inline darwin/client.py]: So my personal opinion is that these prints shouldn't be there by default, but there should be a flag that would be checked in order to figure out if printing debug information. Something like `--debug` or `--verbose`

Comment (Renthal) [reply darwin/client.py]: Previous version was printing a TODO. I would suggest that work on that on a specific PR which addresses failed post/put/get requests if you agree.

Comment (andreaazzini) [reply darwin/client.py]: It's ok if we deal with it in a separate PR, but let's not print stuff here then, when it could lead to data leaking

Comment (Renthal) [reply darwin/client.py]: Agreed. Commented out.

**Pull request merged**